### PR TITLE
Add internal server component

### PR DESCRIPTION
**Pull request by emdv88**

### CODE REVIEW

File: components/internal_server/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# *Quality-time* external server
```
Comment (fniessink) [inline components/internal_server/README.md]:
```suggestion
# *Quality-time* internal server
```

File: components/internal_server/requirements/requirements-dev.in [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-r ../../../requirements/requirements-dev.in
+-r ../../../requirements/types-requests.in
+types-cryptography==3.3.15
```
Comment (fniessink) [inline components/internal_server/requirements/requirements-dev.in]: Why does the internal server need types-cryptography?

File: components/internal_server/requirements/requirements.in [hunk 3]
```diff
@@ -0,0 +1,4 @@
+-r ../../../requirements/bottle.in
+-r ../../../requirements/lxml.in
+gevent==21.12.0
+pymongo==4.1.1
```
Comment (fniessink) [inline components/internal_server/requirements/requirements.in]: The external server needs gevent and pymongo too, maybe put them in ../../../requirements/gevent.in and ../../../requirements/pymongo.in, respectively?

File: components/internal_server/setup.cfg [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+warn_unused_ignores = true
+
+[mypy-bottle]
+ignore_missing_imports = true
+
+[mypy-gevent]
+ignore_missing_imports = true
+
+[mypy-lxml.html.clean]
+ignore_missing_imports = true
```
Comment (fniessink) [inline components/internal_server/setup.cfg]: Does the internal server use lxml?

**Review by fniessink — CHANGES_REQUESTED** [review]

File: components/internal_server/setup.cfg (hunk 4 above)
Comment (emdv88) [reply components/internal_server/setup.cfg]: I don't know, it is still in the requirements. But I wouldn't know how to check 100% certain if we still need it.

File: components/internal_server/requirements/requirements.in (hunk 3 above)
Comment (emdv88) [reply components/internal_server/requirements/requirements.in]: You use these requirements files to ensure identical versions?

File: components/internal_server/requirements/requirements-dev.in (hunk 2 above)
Comment (emdv88) [reply components/internal_server/requirements/requirements-dev.in]: it doesn't, good point

File: components/internal_server/requirements/requirements.in (hunk 3 above)
Comment (fniessink) [reply components/internal_server/requirements/requirements.in]: Yes, and it's an attempt to reduce duplication among the requirements files.

**Pull request merged**